Add a checklist of administrator responsibilities (#43)
redhat-documentation/supplementary-style-guide: ADMIN.adoc

= Repository Administrator Responsibilities

== Administrator rotation schedule:

Every 2 weeks a different project member assumes the role of repository administrator.
Signing up to be administrator is voluntary.

== Administrator responsibilities:

> As the administrator, you are in charge of performing the following duties:

Reviewer: ```suggestion
As the administrator, you are the point of contact if any contributors experience issues with the repository.

In addition, you are in charge of the following:
* Maintaining the Style Guide site on GitHub
* Driving resolution of open issues
* Driving completion of pull requests
```

Reviewer 2: Instead of "In addition, you are in charge of the following:" in @stoobie's suggested change, how about "You also have the following responsibilities:"

> === Being the Point of Contact for general issues related to the Supplementary Style Guide repository

Reviewer 3: Maybe point of contact should be lowercase?

Reviewer 4: +1

Author: It should:) Fixed. Thank you:)

> * You are the point of contact in case other contributors experience issues with the repository.

Reviewer 2: ```suggestion
* You are the point of contact if contributors experience issues with the repository.
```

Reviewer 2: Being pedantic now rather than someone pointing out these minor issues down the line.

When people reach out to you about an issue, try to fix it. If you cannot fix the issue, report it to a member of the initiative who can.

> === Maintaining the Style Guide site on GitHub

Reviewer 4: I would also recommend updating the instruction to flag the issue with "Needs Fix" (we can add that if people agree) and simply @ the admin group (redhat-documentation/ccs-style-council)

> === Maintaining the Style Guide site on GitHub

Author: > @inoxx03 Thanks a lot for drafting these responsibilities! Just a few minor comments.
> 
> What do you think about including a section to this page from the README page? Otherwise people might not know that it exists.

@bergerhoffer I think that is a good point. Let me add the link after I incorporate the collective feedback form project members. Thank you.

* Ensure that the link:https://redhat-documentation.github.io/supplementary-style-guide/[style guide site] is available and displayed correctly.

> * Create pull requests to update the link:https://github.com/redhat-documentation/supplementary-style-guide/blob/master/README.md[README.md], link:https://github.com/redhat-documentation/supplementary-style-guide/blob/master/GUIDELINES.adoc[GUIDELINES.adoc] and the link:https://github.com/redhat-documentation/supplementary-style-guide/blob/master/CONTRIBUTING.md[Contribution guide], if necessary.

Author: Good point! Thank you. Changed

> === Driving resolution of link:https://github.com/redhat-documentation/supplementary-style-guide/issues[open issues]

Reviewer: I don't think there should be links in headings. You already have this link in the first bullet.

In general, I think that having duplicate links in the same topic is not user-friendly.

Reviewer 2: +1

Author: @stoobie @JoanneOFlynn2018 Got it, I moved the links form headings to regular text and I removed the duplicates.

* Review link:https://github.com/redhat-documentation/supplementary-style-guide/issues[issues] submitted to the Supplementary Style Guide GitHub repository.
* Ensure that discussion on open issues is moving forward consistently. If the discussion does not progress, notify link:https://github.com/orgs/redhat-documentation/teams/ccs-style-council[CCS Style Council] members and report any issues that need your peer's attention at the next meeting.

> * Update issue priority labels according to decisions made in meetings.

Reviewer 5: I would probably remove this, I think this might confuse people more than help. We don't really have priority labels (like low, medium, high). The only thing we have really is the "on hold", which I don't think we need admin rotation members to worry about adding.

Reviewer 4: +1.

Author: @bergerhoffer That's reasonable. Thank you. I removed the line per your suggestion:)

* Close issues that are resolved.

=== Driving completion of link:https://github.com/redhat-documentation/supplementary-style-guide/pulls[pull requests]

> 

Reviewer: * Review open link:https://github.com/redhat-documentation/supplementary-style-guide/pulls[pull requests].

* Ensure that contributors are submitting pull requests on prioritized open issues.
* Ensure that work on open pull requests is moving forward consistently.
* Ensure that at least three members of the link:https://github.com/orgs/redhat-documentation/teams/ccs-style-council[CCS Style Council] review and approve every contribution before merging it. You can be one of the three reviewers.
* If a contribution is not reviewed or approved for longer than 1 week, reach out to individual Style Council members and ask them to review and approve the contribution.

> * Merge pull requests with contributions that are approved by three or more Style Council members.

Reviewer 4: @bergerhoffer @inoxx03 Style Council is also just an informal name that doesn't have any defined meaning outside of CCS, so maybe just referring to "project members" is best across this doc.

> * Merge pull requests with contributions that are approved by three or more Style Council members.

Author: @cbudz @bergerhoffer Sounds good. Replaced references to _CCS style council_ with the more generically-worded term _project members_.

= Repository administrator responsibilities

== Administrator rotation schedule

Every 2 weeks a different project member assumes the role of repository administrator.
Signing up to be an administrator is voluntary.

> To volunteer to be an administrator, contact @cbudz or @bergerhoffer.

Reviewer 5: Would you mind making these links to https://github.com/cbudz and https://github.com/bergerhoffer?

> 

Reviewer 4: @inoxx03 please insert at the end of this section that each administration rotation is marked on the Style Guide shared calendar, and admins will receive a notification and calendar invite when it's their turn.

== Administrator responsibilities

As the administrator, you are the point of contact if any contributors experience issues with the repository.
You also have the following responsibilities:

* Driving resolution of open issues
* Driving completion of pull requests
* Maintaining the Style Guide site on GitHub

=== Driving resolution of open issues

* Review link:https://github.com/redhat-documentation/supplementary-style-guide/issues[issues] submitted to the supplementary style guide GitHub repository.

> * If none of the project members respond to a newly opened issue, add the _Needs Fix_ label to the issue and mention @redhat-documentation/ccs-style-council in a comment on the issue to notify project members about it.

Author: > I would also recommend updating the instruction to flag the issue with "Needs Fix" (we can add that if people agree) and simply @ the admin group (redhat-documentation/ccs-style-council)

@cbudz I added a note about the _Needs Fix_ label, but was not sure where you wanted me to place it. Is it OK if I include it under _Driving resolution of open issues_?

Reviewer: @inoxx03 That does look like a logical place to me.

Reviewer 4: @inoxx03 that works for me.

> * If none of the project members respond to a newly opened issue, add the _Needs Fix_ label to the issue and mention @redhat-documentation/ccs-style-council in a comment on the issue to notify project members about it.

Reviewer 5: Maybe make @redhat-documentation/ccs-style-council a link to https://github.com/orgs/redhat-documentation/teams/ccs-style-council?

Reviewer 5: Also, what is the "Needs fix" label for? I see that we don't have this label and I'm not sure I understand its purpose here. CC @cbudz

Reviewer 5: After discussing, the Needs fix label isn't needed. CC @inoxx03

Author: Thank you @bergerhoffer. I'll update the PR and remove the line about the _Needs Fix_ label.

//* Ensure that discussion on open issues is moving forward consistently. If the discussion does not progress, notify link:https://github.com/orgs/redhat-documentation/teams/ccs-style-council[project council members] about the issue.

> * Close issues that are resolved.

Reviewer 4: suggestion: "include a link to the PR that resolves the issue, if one exists."

= Repository administrator responsibilities

== Administrator rotation schedule

Every 2 weeks a different project member assumes the role of repository administrator.
Signing up to be an administrator is voluntary.

> To volunteer to be an administrator, contact link:https://github.com/cbudz[@cbudz] or link:https://github.com/cbudz[@bergerhoffer].

Reviewer 5: The second link needs to be updated to be bergerhoffer

Reviewer 6: @inoxx03 Would you be able to push a new commit with Andrea's requested change (earlier)?

Thanks, --Greg

Author: @gregraka Thank you for spotting this! My bad. Fixed in 6cd6d68
CC @bergerhoffer

> To volunteer to be an administrator, contact link:https://github.com/cbudz[@cbudz] or link:https://github.com/cbudz[@bergerhoffer].

Author: ```suggestion
To volunteer to be an administrator, contact link:https://github.com/cbudz[@cbudz] or link:https://github.com/bergerhoffer[@bergerhoffer].
```
